config/routes.rb

```diff
@@ -3,6 +3,6 @@
   devise_for :users
 
   resources :airlines do
-    resources :reviews, only: [:index, :new, :create, :update, :show]
+    resources :reviews, only: [:index, :new, :create, :update, :show, :edit, :destroy]
```

Comment:
When you have all the resources you no longer need to pass the :only block, get rid of it